src/components/TimerControls.tsx

```diff
@@ -9,9 +12,15 @@ const TimerControls = ({
 }): JSX.Element => {
 	return (
 		<>
-			<button type="button" onClick={pauseTimer}>
-				{isTimerPaused ? "Resume" : "Pause"}
-			</button>
+			{isTimerPaused ? (
```

Comment:
Line 15 to 23 (everything to do with the PauseButton) can probably be moved into another file/component to make it cleaner.

In that way, you can use the following here 

```
<PauseResumeButton onClick={pauseTimer} />
```

Dont forget to also move the code in `TimerControls.styled.tsx` to the new styled file.